functions/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "functions",
+  "description": "Cloud Functions for Firebase",
+  "scripts": {
+    "lint": "eslint .",
+    "serve": "firebase emulators:start --only functions",
+    "shell": "firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "8"
+  },
+  "dependencies": {
+    "@google-cloud/logging-winston": "^3.0.6",
+    "@hapi/joi": "^17.1.1",
+    "config": "^3.3.1",
+    "debug": "^4.1.1",
```

Comment:
are we using these both?

Reply:
Debug, no. Config we are not using right now. I just added in case we have some configuration to use in future.